ui/app/serializers/azure/config.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import ApplicationSerializer from '../application';
+
+export default class AzureConfigSerializer extends ApplicationSerializer {
+  normalizeResponse(store, primaryModelClass, payload, id, requestType) {
+    if (!payload.data) {
+      return super.normalizeResponse(...arguments);
+    }
+    // remove rootPasswordTtl and identityTokenTtl if the API's default value of 0. We don't want to display this value on configuration details if they haven't changed the default value
```

Comment:
Interesting! We've never filtered this out before, is this a new, design request? For what it's worth, `/sys/internal/ui/mounts` returns the actual default value. I remember this because I added it for auth methods [here](https://github.com/hashicorp/vault/blob/main/ui/app/adapters/auth-method.js#L24-L25)